### PR TITLE
Fix population typo [VS-1155]

### PR DESCRIPTION
**Pull request by rsasch**
Created VAT here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/b5c03ce6-cc74-462a-b81d-8ea102be314e
Validated VAT here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/e6511960-8340-4e74-8f06-6c1a69d848bd (confirming with Rori that the failures are not surprising given it's only 10 samples)

### CODE REVIEW

File: scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl [hunk 1]
```diff
@@ -349,22 +349,22 @@ task SchemaNoNullRequiredFields {
 
         # non-nullable fields: vid, contig, position, ref_allele, alt_allele, gvs_all_ac, gvs_all_an, gvs_all_af, variant_type, genomic_location
 
-        bq --apilog=false query --nouse_legacy_sql --project_id=~{project_id} --format=csv
+        bq --apilog=false query --nouse_legacy_sql --project_id=~{project_id} --format=csv \
```
Comment (mcovarr) [inline scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl:352]: uh how could this have worked before?

File: scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl [hunk 2]
```diff
@@ -488,7 +488,7 @@ task SchemaPrimaryKey {
 
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
 
-        bq --apilog=false query --nouse_legacy_sql --project_id=~{project_id} --format=csv
+        bq --apilog=false query --nouse_legacy_sql --project_id=~{project_id} --format=csv \
```
Comment (mcovarr) [inline scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl:491]: same q

File: scripts/variantstore/wdl/extract/extract_subpop.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
  "eas",
  "eur",
  "mid",
+ "nfr",
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/extract_subpop.py:10]: I thought this was PR was about changing nfr to nfe (or at least that's what the ticket says)

**Review by mcovarr — CHANGES_REQUESTED** [review]
I think this is supposed to be changing this to NFE, not NFR

File: scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl:352]: dunno, but it was breaking without it

**Review by mcovarr — APPROVED** [review]
code changes look fine but obvs we want to root cause the validation failure

**Pull request merged**